central/complianceoperator/v2/scans/datastore/datastore_impl.go

```diff
@@ -0,0 +1,64 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/complianceoperator/v2/scans/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+)
+
+var (
+	complianceSAC = sac.ForResource(resources.Compliance)
```

Comment:
switched to Compliance SAC.

Reply:
We're handling the cluster scoping portion separately right?

Reply:
yeah switching to resource.Compliance makes it cluster scoped and as such the queries and searches deal with it for the most part.  I am going to add some tests while I'm in here though.